tests/types.test.tsx

```diff
@@ -1,6 +1,6 @@
 import { expectType } from 'ts-expect'
-import { atom, useAtom } from 'jotai'
 import type { Atom, PrimitiveAtom, WritableAtom } from 'jotai'
+import { atom, useAtom } from 'jotai'
```

Comment:
let's revert this if it's not intentional.

Reply:
Whoops, good catch. Fixed.